src/styles/common.scss

```diff
@@ -236,6 +236,37 @@
     margin-bottom: 1%;
 }
 
+@media (max-width: $desktop-width) {
+    .textBanner {
+        padding: 70px 14%;
+    }
+
+    .textBannerSubtitle {
+        font-size: 1.8vw;
+        margin-bottom: 2%;
+    }
+}
+
+@media (max-width: $tablet-width) {
+    .textBanner {
+        padding: 70px 0;
+    }
+
+    .textBannerSubtitle {
+        font-size: 14px;
+    }
+
+    .textBannerTitle {
+        font-size: 6.3vw;
+    }
+}
+
+@media (max-width: $tablet-width) {
```

Comment:
You already have a tablet width media query above.

Reply:
I see it. Fixed